tests/test_solvers/test_harbor_tmux_patch.py

```
@@ -0,0 +1,192 @@
+
+# Skip the whole module unless Harbor's terminus-2 TmuxSession is importable.
+_tmux_mod = pytest.importorskip("harbor.agents.terminus_2.tmux_session")
+TmuxSession = _tmux_mod.TmuxSession
+
+LEAN4_PROOF_STEP33_FIXTURE = Path(__file__).parent / "fixtures" / "lean4_proof_repeat0_step33_keystrokes.txt"
+
+SESSION_NAME = "test-session"
+
+
+@pytest.fixture(autouse=True)
+def _apply_patch():
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_apply_patch`

Add return type annotation: `None`

(ANN202)